<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testmode</class>
 <widget class="QWidget" name="testmode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbTolColi">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>总大肠菌群</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbFecColi">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>耐热大肠菌群</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEColi">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>大肠埃希氏菌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTPC">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>细菌总数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>410</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>531</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>400</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>快速检测</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>pbTPC</zorder>
  <zorder>pbTolColi</zorder>
  <zorder>pbFecColi</zorder>
  <zorder>pbEColi</zorder>
  <zorder>pbExit</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
